dockers/screenshotter/screenshotter.js

```diff
@@ -153,7 +154,10 @@ function guessDockerIPs() {
     }
     // Native Docker on Linux or remote Docker daemon or similar
     const gatewayIP = cmd("docker", "inspect",
-      "-f", "{{.NetworkSettings.Gateway}}", opts.container);
+        "-f", "{{.NetworkSettings.Gateway}}", opts.container)
+      || cmd("docker", "inspect",
+        "-f", "{{range .NetworkSettings.Networks}}{{.Gateway}}{{end}}",
+        opts.container);
```

Comment:
Why do we need two different commands now?

Reply:
@kevinbarabash If Docker is configured to use the non-default network interface, the IP address of the gateway is available at not `.NetworkSettings.Gateway` but `.NetworkSettings.Networks.*.Gateway`.

Reply:
Can you add a comment saying that here before merging this PR?